src/dto/daily-calibration.dto.ts

```diff
@@ -42,22 +46,31 @@ export class DailyCalibrationBaseDTO {
 
   @IsOptional()
   @IsNumber()
+  @Min(0)
+  @Max(23)
   upscaleInjectionHour?: number;
 
   @IsOptional()
   @IsNumber()
+  @Min(0)
+  @Max(59)
   upscaleInjectionMinute?: number;
 
   @IsOptional()
   @IsNumber()
+  @Min(0)
+  @Max(10000000000)
   zeroMeasuredValue?: number;
 
   @IsOptional()
   @IsNumber()
+  @Min(0)
+  @Max(10000000000)
   upscaleMeasuredValue?: number;
 
   @IsOptional()
   @IsNumber()
+  @IsIn([0, 1])
   zeroAPSIndicator?: number;
```

Comment:
Does @IsIn([0,1]) need to be added for upscaleAPSIndicator as well?

Reply:
Updated